packages/glimmer-demos/lib/ripples.ts

```diff
@@ -79,18 +79,14 @@ export function start() {
     data.points.push( new Point(e.timeStamp, e.offsetX, e.offsetY) );
   });
 
-  let baseOffset = Date.now() - window.performance.now();
```

Comment:
Thanks for the details. `e.timeStamp` seems to be in a particularly bad state:

 * In chrome 49, it will be something like `11456` which is ms since page load. 
 * In firefox 45, it will be something like `9362308` which is ms since machine start. ([issue tracked here](https://bugzilla.mozilla.org/show_bug.cgi?id=238041))
 * In safari 9, it will be something like `1459630789270` which is ms since 1st Jan 1970.

Some options:

 1. use `new Date().getTime()` instead of `e.timeStamp` which will work on all browsers but will be less accurate
 2. use a normalised `e.timeStamp` for Chrome and Safari and `new Date().getTime()` for Firefox
 3. use a normalised `e.timeStamp` for Chrome and Safari and drop support for Firefox (with a message)

How important is accuracy? Should I go with option 1)?